crates/optimism/evm/src/config.rs

```diff
@@ -0,0 +1,133 @@
+use reth_chainspec::{ChainSpec, OptimismHardfork};
+use reth_ethereum_forks::{EthereumHardfork, Head};
+
+/// Returns the spec id at the given timestamp.
+///
+/// Note: This is only intended to be used after the merge, when hardforks are activated by
+/// timestamp.
+pub fn revm_spec_by_timestamp_after_merge(
+    chain_spec: &ChainSpec,
+    timestamp: u64,
+) -> revm_primitives::SpecId {
+    if chain_spec.fork(OptimismHardfork::Fjord).active_at_timestamp(timestamp) {
+        revm_primitives::FJORD
+    } else if chain_spec.fork(OptimismHardfork::Ecotone).active_at_timestamp(timestamp) {
+        revm_primitives::ECOTONE
+    } else if chain_spec.fork(OptimismHardfork::Canyon).active_at_timestamp(timestamp) {
+        revm_primitives::CANYON
+    } else if chain_spec.fork(OptimismHardfork::Regolith).active_at_timestamp(timestamp) {
+        revm_primitives::REGOLITH
+    } else {
+        revm_primitives::BEDROCK
+    }
+}
+
+/// return `revm_spec` from spec configuration.
+pub fn revm_spec(chain_spec: &ChainSpec, block: Head) -> revm_primitives::SpecId {
```

Comment:
I'm pretty sure we no longer need this function because we can only support post-bedrock anyway

is this correct @clabby ?